content/applications/productivity/calendar/google.rst

```diff
@@ -153,3 +153,60 @@ Now, Odoo Calendar is successfully synced with Google Calendar!
 
    Events can be created in Google Calendar without sending a notification by selecting
    :guilabel:`Don't Send` when prompted to send invitation emails.
+
+Google OAuth FAQ
+================
+
+At times there can be misconfigurations that take place, and troubleshooting is needed to resolve
+the issue. Below are the most common errors that may occur when configuring the Google Calendar for
```

Comment:
```suggestion
the issue. Below are the most common errors that may occur when configuring the *Google Calendar* for
```
Since, in theory, *Google Calendar* is an app -- I'd suggest italicizing every mention of it, in order to stay in line with our formatting rules regarding the italicization of applications/software